generators/client/templates/angular/src/main/webapp/app/account/sessions/_sessions.service.ts

```diff
@@ -30,10 +30,10 @@ export class SessionsService {
     constructor(private http: Http) { }
 
     findAll(): Observable<Session[]> {
-        return this.http.get(this.resourceUrl).map((res: Response) => res.json());
+        return this.http.get<Session[]>(this.resourceUrl);
```

Comment:
I really liked this one 👍 It clearly states what is gonna be there ❤️

Reply:
i tried to type everything i stubled over not sure though if i really did it eveywhere so far